packages/core/utils/lib/convert-query-params.js

```diff
@@ -250,43 +250,44 @@ const convertAndSanitizeFilters = (filters, schema) => {
 
   // Here, `key` can either be an operator or an attribute name
   for (const [key, value] of Object.entries(filters)) {
-    const attribute = get('key', schema.attributes);
-
-    // Handle attributes
-    if (attribute) {
-      // Relations
-      if (attribute.type === 'relation') {
-        filters[key] = convertAndSanitizeFilters(value, strapi.getModel(attribute.target));
-      }
-
-      // Components
-      else if (attribute.type === 'component') {
-        filters[key] = convertAndSanitizeFilters(value, strapi.getModel(attribute.component));
-      }
+    if (schema && Object.prototype.hasOwnProperty.call(schema, 'attributes')) {
```

Comment:
`schema` should always be defined there (thus always having an `attributes` field), so I think you can remove that check. If for any reason we end up with an `undefined` schema here, it should break here & throw instead of silently pass.